zerver/lib/generate_test_data.py

```diff
@@ -27,12 +27,17 @@ def generate_topics(num_topics: int) -> List[str]:
     for _ in itertools.repeat(None, num_single_word_topics):
         topics.append(random.choice(config["nouns"]))
 
+    # Make embed link topics account for 5% of total topics.
+    num_embed_link_topics = num_topics // 20
```

Comment:
I think this will be 0 most of the time; maybe a `min(1, )` would be a good idea.